### PR TITLE
Styling of Events

### LINKED ISSUE #1103: Clean Up Events Page Styling and Ordering
### Overview
The events page for logged in admins isn't presenting events in an organized way. Restyle and reorganize it to be similar to  https://www.hackforla.org/events/

### Action Items

- [x] #1130
- [x]  Order events by time.
- [ ] Clean up styling to be similar to the events list on the vrms website (adjusted to fit the width of our vrms interface)
- [ ] Display the same data as the events list on the website
- [x] ~Get rid of details link and page~ - Removed this requirement. The details will now be useful once the edit button is added
- [ ] Remove add event button
- [ ] Adding loading spinner/state
- [ ] Add simple search/filter by Event name and project name to the page (just on the front-end)

### PR DESCRIPTION
**Pull request by evanyang1**
Closes #1103: all tasks resolved except for the part where the events have to be sorted by day of week.

### CODE REVIEW

File: client/src/sass/Events.scss [hunk 1]
```diff
@@ -9,9 +9,9 @@
     -webkit-box-pack: justify;
         -ms-flex-pack: justify;
             justify-content: space-between;
-    padding: 6px 6px 6px 6px;
-    margin: 0px 6px 18px 6px;
-    border-bottom: 2px solid #FAFAFA;
+    //padding: 2px 2px 2px 2px;
```
Comment (FoxShaunR) [inline client/src/sass/Events.scss]: Please remove this instead of commenting it out.

File: client/src/pages/Events.js [hunk 2]
```diff
@@ -36,21 +37,34 @@ const Events = (props) => {
...
+                <div key={index} className="list-event-container">
+                  <div className="list-event-headers">
+                    <p className="event-name">
+                      {' '}
+                      <Link to={`/event/${event._id}`}>{event.name}</Link> (
+                      {moment(event.date).format('ddd, MMM D @ h:mm a')})
+                    </p>
 
-                  <div className="event-info">
+                    {/* <div className="event-info">
```
Comment (FoxShaunR) [inline client/src/pages/Events.js]: Remove this instead of commenting out

File: client/src/pages/Events.js [hunk 3]
```diff
@@ -36,21 +37,34 @@ const Events = (props) => {
...
       <ul>
-        {events.map((event, index) => {
-          const event_city = event.location && (event.location.city || 'TBD');
-          const event_state = event.location && (event.location.state || 'TBD');
+        {events
+          .filter((event) => {
+            console.log(typeof event.name);
+            return typeof event.name === 'string' && event.name.toLowerCase().match(eventSearchParam.toLowerCase());
+          })
+          //.filter((event) => event.name.match(eventSearchParam))
```
Comment (FoxShaunR) [inline client/src/pages/Events.js]: hehe, this comment too please

File: client/src/pages/Events.js [hunk 4]
```diff
@@ -36,21 +37,34 @@ const Events = (props) => {
...
+        value={eventSearchParam}
+        onChange={(e) => setEventSearchParam(e.target.value)}
+      />
       <ul>
-        {events.map((event, index) => {
-          const event_city = event.location && (event.location.city || 'TBD');
-          const event_state = event.location && (event.location.state || 'TBD');
+        {events
+          .filter((event) => {
+            console.log(typeof event.name);
```
Comment (FoxShaunR) [inline client/src/pages/Events.js]: Remove this console.log now that we don't need it for debugging

**Comment by FoxShaunR** [comment]
Split off the remaining item to https://github.com/hackforla/VRMS/issues/1130 @jbubar Feel free to merge and close the parent ticket once you're done reviewing

**Review by jbubar — APPROVED** [review]
This is great evan!

**Pull request merged**